src/release/triage-procedure.md

```diff
@@ -51,9 +51,33 @@ not need a review yet.)
 
 ### Procedure
 
-We primarily triage three status labels: S-waiting-on-review,
-S-waiting-on-author, and (once in a while) S-blocked. Here is the procedure for
-each:
+We primarily triage Pull Requests with three status labels: `S-waiting-on-review`,
+`S-waiting-on-author`, and (once in a while) `S-blocked`.
+
+#### Checking for contributor activeness
+
+Some contributors are on leave (permanently or temporarily), and it can be quite
+annoying to get pinged (or can make them feel pressured otherwise, even if that's
+not the intention of the person doing PR triaging). We should **avoid pinging
+contributors who are on leave or inactive**.
+
+To check if a contributor is active or not, we can:
+
+- Check if contributor is in alumni status. If a contributor is alumni status, they
+  should be considered inactive. This can be checked by searching [rust-lang/team].
+  You can use a GitHub search query like
+  `repo:rust-lang/team path:/teams/ {GITHUB_HANDLE}` (with `GITHUB_HANDLE` being
+  the contributor).
```

Comment:
Maybe worth noting that we never ping people on `T-*` teams either, since they'll also get that info from this search?

Reply:
Can you elaborate what you mean by

> never ping people on `T-*` teams?

Reply:
I believe we have a guideline to not ping authors if they are on a team, and the instructions don't say anything about pinging reviewers as far as i can see.